app/services/katello/product_content_importer.rb

```diff
@@ -100,6 +102,12 @@ def import
     private def update_content(content, prod_content_json)
       new_name = prod_content_json[:content][:name]
       content.update_attributes!(:name => new_name) if content.name != new_name
+
+      new_url = prod_content_json[:content][:contentUrl]
```

Comment:
It looks like if name changes and url changes that would issue two update queries - so I think refactoring this to run a single query may be better

Reply:
updated